charts/gardener-extension-admission-aws/values.yaml

```diff
@@ -32,6 +32,9 @@ global:
         -----BEGIN RSA PRIVATE KEY-----
         ...
         -----END RSA PRIVATE KEY-----
+    # objectSelector:
+    #   matchLabels:
+    #     provider.extensions.gardener.cloud/aws: "true"
```

Comment:
What about a simple flag `useObjectSelector` which is `true` or `false`? Then operators would not need to specify them here.